pyccel/codegen/python_wrapper.py

```diff
@@ -129,8 +130,16 @@ def create_shared_library(codegen,
 
         if verbose:
             print(' '.join(cmd))
-        p = subprocess.Popen(cmd, stdout=subprocess.PIPE, stderr=subprocess.PIPE, universal_newlines=True)
-        out, err = p.communicate()
+        lock_names = [d+'.lock' for d in dep_mods]
+        locks = [FileLock(n) for n in lock_names]
```

Comment:
Could you combine these two lines as well?

Reply:
Fixed in 2a4be0b